tests/integration/targets/eos_lacp_interfaces/tests/common/merged.yaml

```diff
@@ -5,10 +5,10 @@
     config:
 
       - name: Ethernet1
-        rate: fast
```

Comment:
You might want to add an additional case keeping the alias "rate" to make sure both work as expected.

Reply:
The alias is tested in the unit test suit.